scripts/deploy.sh

```diff
@@ -134,11 +136,13 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
       echo "apply_security_hardening: false" >> $OA_OVERRIDES
     fi
 
+    # set the affinity to 3 for infra cluster (necessary for maas testing)
+    awk '{print} $0 == "  aio1:" && previous == "shared-infra_hosts:" \
+         {print "    affinity:\n      galera_container: 3\n      rabbit_mq_container: 3"} \
+         {previous = $0}' \
+    /etc/openstack_deploy/openstack_user_config.yml > /tmp/tmp && mv /tmp/tmp /etc/openstack_deploy/openstack_user_config.yml
```

Comment:
@hughsaunders: ``galera_container`` and ``rabbit_mq_container`` no longer exist in ``openstack_user_config.yml`` so I don't think the above command will help. What I am doing with the awk bits is adding ``galera_container: 3`` and ``rabbit_mq_container: 3`` after these lines:
```
aio1:
  shared-infra_hosts:
```